mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -7705,7 +7715,60 @@ resource "google_compute_instance" "foobar2" {
   }
 
 }
-`, instance, confidentialInstanceType, instance)
+
+<% unless version == "ga" -%>
+resource "google_compute_instance" "foobar3" {
+  name         = "%s3"
+  machine_type = "c3d-standard-4"
+  zone         = "us-central1-a"
+
+  boot_disk {
+    initialize_params {
+      image = data.google_compute_image.my_image.self_link
+    }
+  }
+
+  network_interface {
+    network = "default"
+  }
+
+  confidential_instance_config {
+    enable_confidential_compute       = true
+    confidential_instance_type        = %q
+  }
+
+  scheduling {
+	  on_host_maintenance = "TERMINATE"
+  }
+
+}
+
+resource "google_compute_instance" "foobar4" {
+  name         = "%s4"
+  machine_type = "c3d-standard-4"
+  zone         = "us-central1-a"
+
+  boot_disk {
+    initialize_params {
+      image = data.google_compute_image.my_image.self_link
+    }
+  }
+
+  network_interface {
+    network = "default"
+  }
+
+  confidential_instance_config {
+    enable_confidential_compute       = true
+  }
+
+  scheduling {
+    on_host_maintenance = "TERMINATE"
+  }
+
+}
+<% end -%>
+`, instance, confidentialInstanceType, instance, instance, confidentialInstanceType, instance)
```

Comment:
Won't this have too many arguments in the GA case? I think you need an `else` clause with the old arguments.

Reply:
Done, thanks.